core/jazz_ui/e2e/07-slsappservices/slsappservices.e2e-spec.ts

```diff
@@ -0,0 +1,451 @@
+ */
+
+import { browser, element, by, protractor, $ } from 'protractor';
+import { Jazz } from '../page-objects/jazzservices.po';
+import { Timeouts, Browser } from 'selenium-webdriver';
+import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG, SSL_OP_TLS_BLOCK_PADDING_BUG } from 'constants';
+import { Common } from '../common/commontest';
+
+
+xdescribe('Overview', () => {
+  let jazzServices_po: Jazz;
+  let commonUtils: Common;
+  let found = 1;
+  let flag = 1;
+  const EC = protractor.ExpectedConditions;
+  let winhandle;
+  let servicename;
+
+  beforeAll(() => {
+    jazzServices_po = new Jazz();
+    commonUtils = new Common();
+    
+  });
+  beforeEach(() => {
+    if (flag == 0) {
+      pending();
+    }
+    // if (found == 0) {
+    //   commonUtils.fluentwaittry(jazzServices_po.getServiceHomePage(), Common.shortWait);
+    //   jazzServices_po.getServiceHomePage().click();
+    // }
+    if (found == 0){
+        pending();
+    }
+  });
+  afterAll(() => {
+    jazzServices_po = new Jazz();
+    commonUtils = new Common();
+    browser.driver.sleep(Common.miniWait);
+    jazzServices_po.logoutIcon().click();
+    jazzServices_po.logout().click();
+  });
+
+
+  function createservice(servicename) {
+    jazzServices_po.getServiceName().sendKeys(servicename);
+    jazzServices_po.getNameSpace().sendKeys('jazztest');
+    jazzServices_po.getServiceDescription().sendKeys('Testing');
+  }
+
+  function serviceapprover() {
+    browser.driver.sleep(Common.miniWait);
+    jazzServices_po.getSubmit().click();
+    jazzServices_po.getDone().click().then(null, function (err) {
+      flag = 0;
+    });
+    commonUtils.waitForSpinnerLogin();
+  }
+
+  function waitforskiptest(ele, t) {
+    browser.manage().timeouts().implicitlyWait(0);
+    browser.wait(function () {
+      browser.sleep(t);
+      return ele.isDisplayed()
+        .then(
+          function (text) {
+            flag = 1;
+            return text;
+          },
+          function (error) {
+            browser.refresh();
+            flag = 0;
+            return false;
+          });
+    }, 240 * 1000);
+  }
+
+  it('Create SLS-App Service', function () {
+    browser.driver.sleep(Common.miniWait);
+    browser.wait(EC.visibilityOf(jazzServices_po.getCreateService()), Common.timeOutHigh).then(null, function (err) {
+      console.log(err);
+      flag = 0;
+      browser.refresh();
+    });
+    browser.wait(EC.elementToBeClickable(jazzServices_po.getCreateService()), Common.timeOutHigh);
+    jazzServices_po.getCreateService().click();
+    browser.driver.switchTo().activeElement();
+    browser.driver.sleep(Common.miniWait);
+    //Creating Website
+    if (!jazzServices_po.getBuildCustom()){
+        flag = 0;
+    }else{
+        jazzServices_po.getBuildCustom().click();
+    }
+    var min = 111111111;
+    var max = 999999999;
+    var randomNum = Math.floor(Math.random() * (max - min + 1)) + min;
+    servicename = 'servicename' + randomNum;
+    createservice(servicename);
```

Comment:
please add a deployment descriptor for sls app. I believe without one it may fail

Reply:
deployment descriptor for sls app is removed in new UI changes